### PR TITLE
Updates for 1B servicing release

### PR DESCRIPTION
**Pull request by mthalman**
Related to https://github.com/microsoft/dotnet-framework-docker/issues/871

### CODE REVIEW

File: src/runtime/4.8/windowsservercore-ltsc2022/Dockerfile [hunk 1]
```diff
@@ -9,14 +9,6 @@ ENV `
...
-    # Apply latest patch
-    curl -fSLo patch.msu http://download.windowsupdate.com/d/msdownload/update/software/updt/2021/09/windows10.0-kb5006364-x64-ndp48_b1f07ef46aa26dc6f55d677d156c038a8f5a1686.msu `
-    && mkdir patch `
-    && expand patch.msu patch -F:* `
-    && del /F /Q patch.msu `
-    && dism /Online /Quiet /Add-Package /PackagePath:C:\patch\windows10.0-kb5006364-x64-ndp48.cab `
-    && rmdir /S /Q patch `
-    `
     # Apply patch to provide support for container limits
     && curl -fSLo patch.msu https://download.microsoft.com/download/5/a/4/5a4425d3-eb10-47a8-afe0-b07a19b20b1c/Windows10.0-KB9008395-x64-NDP48.msu `
```
Comment (MichaelSimons) [inline src/runtime/4.8/windowsservercore-ltsc2022/Dockerfile:13]: This doesn't look right to me.  The leading && is going to cause the build to fail.

Comment (mthalman) [reply src/runtime/4.8/windowsservercore-ltsc2022/Dockerfile:13]: Fixed

**Comment by mthalman** [comment]
There's an issue applying KB5009557, the Windows LCU for the 3.5 Dockerfile on Windows Server 2019. I'm reverting the change and continuing the release without that version for now until this can get resolved.

File: src/runtime/4.8/windowsservercore-ltsc2022/Dockerfile (hunk 1 above)
Comment (MichaelSimons) [reply src/runtime/4.8/windowsservercore-ltsc2022/Dockerfile:13]: The fix doesn't look right yet.  The `&&` was removed from the ngen line not the curl line

**Review by MichaelSimons — APPROVED** [review]

**Pull request merged**